File: src/main/java/org/kite9/diagram/visualization/planarization/rhd/grouping/directed/ContainerMergeType.java
package org.kite9.diagram.visualization.planarization.rhd.grouping.directed;

public enum ContainerMergeType { 
	
	WITHIN_LIVE_CONTAINER(0), JOINING_LIVE_CONTAINERS(20), NO_LIVE_CONTAINER(30); 

	
	private int pa;
	
	private ContainerMergeType(int priorityAdjustment) {
		this.pa = priorityAdjustment;
	}

	public int getPriorityAdjustment() {
		return pa;
	}

}

File: visualization/kite9-visualization-common/src/jvmMain/kotlin/org/kite9/diagram/visualization/planarization/rhd/grouping/directed/ContainerMergeType.kt
package org.kite9.diagram.visualization.planarization.rhd.grouping.directed

enum class ContainerMergeType(val priorityAdjustment: Int) {

    WITHIN_LIVE_CONTAINER(0), JOINING_EXTRA_CONTAINERS(20), NO_LIVE_CONTAINER(30);

}